tsconfig.json

```diff
@@ -8,7 +8,7 @@
     "sourceMap": true,
     "declaration": false,
     "module": "es2020",
-    "target": "es2015",
+    "target": "ES2021",
```

Comment:
If we change this to `2022`, we can remove this warning from the dev server:
> TypeScript compiler options "target" and "useDefineForClassFields" are set to "ES2022" and "false" respectively by the Angular CLI. To control ECMA version and features use the Browerslist configuration. For more information, see https://angular.io/guide/build#configuring-browser-compatibility

That said, the first time I built it with ES2022, it took double the time. Can't seem to reproduce it though. Subsequently it was faster than ES2021 by a few seconds per build.

|`target`|Build time|
| --: | --: |
|ES2022|189398ms|
|ES2021|69193ms|
|ES2022|53383ms|
|ES2021|58089ms|
|ES2022|57353ms|
|ES2022|53826ms|

Reply:
There is another bug with v15 with ES2022 and TS. The solution was to downgrade to ES2021. We can update this when we migrate to v16

Reply:
Subsequent builds will always be faster. NG caches some of the build files. That is why the initial build takes a lot more time. Try clearing the cache and rebuilding.